deepinv/training/trainer.py

```diff
@@ -296,7 +296,10 @@ class Trainer:
     verbose: bool = True
     verbose_individual_losses: bool = True
     show_progress_bar: bool = True
-    freq_update_progress_bar: int = 1
+    freq_update_progress_bar: int = 10
+    non_blocking_transfers: bool = (
```

Comment:
Does this have to be an extra argument? if absolutely needed it would need a docstring